src/components/logical-expression-builder/leb-group/leb-group.component.html

```diff
@@ -0,0 +1,64 @@
+<tr class="leb-group" uxHoverActionContainer [style.padding-left.px]="indent">
+    <td>
+        <ux-select
+            [value]="selectedLogicalOperator"
+            (valueChange)="handleSelectedOperatorChange($event)"
+            [readonlyInput]="true"
+            [options]="logicalOperators"
+            [display]="'label'"
+            [key]="'name'">
+        </ux-select>
+    </td>
+    <td>
+        <button
+            uxHoverAction
+            (click)="deleteGroup()"
+            type="button"
+            class="btn btn-link btn-icon btn-lg button-secondary"
+            aria-label="Delete condition">
+            <span class="ux-icon ux-icon-trash text-error" aria-hidden="true"></span>
+        </button>
+    </td>
+</tr>
+
+<ng-container *ngFor="let child of subExpression?.children; index as index">
+    <ux-leb-condition
+        *ngIf="child.type === 'condition'"
+        (conditionChange)="handleGroupChange($event, index)"
+        [condition]="child"
+        [id]="index"
+        (conditionDeleted)="removeConditionAtIndex($event)"
+        (conditionEmbedded)="embedConditionAtIndex($event)"
+    [indent]="indent + 40">
+    </ux-leb-condition>
+
+    <ux-leb-group
+        *ngIf="child.type === 'group'"
+        [subExpression]="child"
+        (groupChange)="handleGroupChange($event, index)"
+        [logicalOperatorName]="child.logicalOperator"
+        [indent]="indent + 40">
+    </ux-leb-group>
+</ng-container>
+
+<tr class="leb-group">
+    <td [style.padding-left.px]="indent + 40">
+        <button
+            [uxMenuTriggerFor]="addMenu"
+            type="button"
+            class="btn btn-icon button-primary"
+            aria-label="Add">
+            <span class="ux-icon ux-icon-add" aria-hidden="true"></span>
+        </button>
+    </td>
+</tr>
+
+<ux-menu #addMenu>
```

Comment:
The AngularJS component shows both field names and logical operators immediately (can be part of next pull request)